docs/building.md

```diff
@@ -120,33 +120,6 @@ the registry binary generated in the "./bin" directory:
     $ ./bin/registry -version
     ./bin/registry github.com/docker/distribution v2.0.0-alpha.2-80-g16d8b2c.m
 
-### Developing
-
-The above approaches are helpful for small experimentation. If more complex
-tasks are at hand, it is recommended to employ the full power of `godep`.
-
-The Makefile is designed to have its `GOPATH` defined externally. This allows
```

Comment:
All this is related to dep management, which is no longer required with go 1.6.
I tend to think removing this was right.
wdyt?